bokeh/charts/chart.py

```diff
@@ -286,7 +286,7 @@ def make_grid(self, dimension, ticker):
             ticker (obj): the axis.ticker object
 
         Return:
-            grid: Grid instance
+            Grid: A Grid instance
```

Comment:
new sphinx versions automatically link to types for args and returns, needed to make it the real type other wise there were problems.